### PR TITLE
Fix pylint errors and add github actions workflow

### PR DESCRIPTION
**Pull request by mayank-singla001**
- Fix the pylint errors for both python2 and python3.
- Add GitHub actions workflows to check the Python files to python2 and python3 compatibility and linting.

### CODE REVIEW

File: BootstrapScriptWithToken/bootstrap.py [hunk 1]
```diff
@@ -206,24 +207,27 @@ def tryImageUpgrade( e ):
...
+   import requests
+   from SysdbHelperUtils import SysdbPathHelper
 except ImportError as e:
    if sys.version_info < (3,) and os.path.exists( '/usr/bin/python3' ):
       os.execl( '/usr/bin/python3', 'python3', os.path.abspath(__file__ ) )
    else:
       log("Python3 not found. Attempting EOS version upgrade")
       tryImageUpgrade( e )
 
-if sys.version_info >= (3,):
```
Comment (harshitk-arista) [inline BootstrapScriptWithToken/bootstrap.py:219]: Why this change? I feel the previous one was 'self-documenting' in some shape.

Comment (mayank-singla001) [reply BootstrapScriptWithToken/bootstrap.py:219]: This will give error with python2, so we have to write it this way for both py2 and py3 compatible.

File: BootstrapScriptWithToken/bootstrap.py [hunk 2]
```diff
@@ -206,24 +207,31 @@ def tryImageUpgrade( e ):
...
+try:
+   # This import will fail when running this script in python2 environment
+   # starting EOS 4.30.1
+   # We can't use the conditional logic here to import the libraries as pylint will
```
Comment (harshitk-arista) [inline BootstrapScriptWithToken/bootstrap.py]: conditional logic does say what we used to do. You can remove the comments from line 223-224. Line 221-222 should be enough.

File: BootstrapScriptWithToken/bootstrap.py [hunk 3]
```diff
@@ -206,24 +207,31 @@ def tryImageUpgrade( e ):
...
 except ImportError as e:
    if sys.version_info < (3,) and os.path.exists( '/usr/bin/python3' ):
       os.execl( '/usr/bin/python3', 'python3', os.path.abspath(__file__ ) )
    else:
       log("Python3 not found. Attempting EOS version upgrade")
       tryImageUpgrade( e )
 
-if sys.version_info >= (3,):
+try:
+   # This import will fail when running this script in python2 environment
```
Comment (harshitk-arista) [inline BootstrapScriptWithToken/bootstrap.py]: The comment should read - This import will fail for EOS < 4.30.1, where #!/usr/bin/python will run this in a python 2 environment

File: BootstrapScriptWithToken/bootstrap.py (hunk 2 above)
Comment (mayank-singla001) [reply BootstrapScriptWithToken/bootstrap.py]: Done

File: BootstrapScriptWithToken/bootstrap.py (hunk 3 above)
Comment (mayank-singla001) [reply BootstrapScriptWithToken/bootstrap.py]: Done

**Review by harshitk-arista — APPROVED** [review]

**Comment by mayank-singla001** [comment]
Closing this as main branch will be synced with dev
